### DISCUSSION TITLE
Decide how to store blog posts

### DISCUSSION THREAD

**Discussion by bradgarropy** [remix-austin/new-remixaustin-com#17] [General]
Here are some quick options off the top of my head.

- markdown files
- cms
- database

What do y'all think? For a blog of this size, which I assume will be small, local markdown files would be the easiest, especially with the ability to glob import with `vite`.

**Comment by bradgarropy** [comment]
@brookslybrand how does the current Remix website handle this?

**Comment by brookslybrand** [comment]
Currently the Remix websites uses the [`docs` directory of the Remix repo](https://github.com/remix-run/remix/tree/main/docs) as the CMS (bunch of md files that get parsed).

Personally, I think that approach is overkill for this project

**Comment by bradgarropy** [comment]
In your opinion, what's the simplest way to handle this?

Does anyone else have thoughts? @sethdavis512 @markmals

**Comment by sethdavis512** [comment]
I like the idea of a blog, please don't get me wrong but I struggle with it because it has been tough getting folks interested in writing posts...We don't have a consistent flow of interest or content.

My vote would be that we put the blog on pause for now. Focus on other pieces (Meetup info, community projects, discord, etc).

Thoughts?

**Comment by bradgarropy** [comment]
Very much agree with you on this @sethdavis512. Let's focus on the community aspects first.

What do you think we should do with this issue in the meantime?

**Comment by sethdavis512** [comment]
Covert this issue to a discussion until we have further interest/more established site?

<img width="492" alt="image" src="https://github.com/remix-austin/new-remixaustin-com/assets/3208705/abb16473-981e-44b5-af8e-a9796fca76d3">

**Reply by bradgarropy** [reply]
Good use of discussions! 👍🏼
